src/python/pants/bin/local_pants_runner.py

```diff
@@ -102,18 +102,11 @@ def create(
         run_tracker = RunTracker(options_bootstrapper.args, options)
         native_engine.maybe_set_panic_handler()
 
-        # Option values are usually computed lazily on demand, but command line options are
-        # eagerly computed for validation.
         with options_initializer.handle_unknown_flags(options_bootstrapper, env, raise_=True):
             # Verify CLI flags.
             if not build_config.allow_unknown_options:
                 options.verify_args()
 
-            for scope, values in options.scope_to_flags.items():
-                if values:
-                    # Only compute values if there were any command line options presented.
```

Comment:
I'm not even sure why this was needed in the past, but we now compute for all scopes in `verify_args()` so this is certainly superfluous now.